.github/CODEOWNERS

```diff
@@ -46,19 +46,19 @@
 /src/pytorch_lightning/tuner                @SkafteNicki @borda @awaelchli
 /src/pytorch_lightning/utilities            @borda @tchaton @carmocca
 
-/src/lightning_app                          @tchaton @manskx
-/src/lightning_app/cli/pl-app-template      @tchaton @awaelchli @Borda
-/src/lightning_app/core                     @tchaton @awaelchli @manskx
-/src/lightning_app/core/queues.py           @tchaton @hhsecond @manskx
-/src/lightning_app/runners/cloud.py         @tchaton @hhsecond
-/src/lightning_app/testing                  @tchaton @manskx
+/src/lightning_app                          @tchaton @manskx @Lightning-AI/lai-platform
+/src/lightning_app/cli/pl-app-template      @tchaton @awaelchli @Borda @Lightning-AI/lai-platform
```

Comment:
I would prefer to have a specific person because as team it may easily happen someone not experienced would gain quite high access level... :rabbit:

Reply:
> We want to spread the knowledge across our team as well so that folks are comfortable supporting this and helping future reviews as well

That is exactly what code-ownership is about, so when some developers become comfortable and responsible for changing some codebase, let's add them :)

Reply:
Hey @narJH27 This is definitely not being merged 😁

The goal is to move toward stability and strong ownership. We won t give merge power to a Team, only indivivual with the right level of expertise.

Reply:
@tchaton @Borda I disagree with this decision to close this PR. We on platform are making developments towards convergence to lai and our development velocity is currently being hampered by not having a process in place where we're able to get reviews from the current codeowners of this repo fast enough. I've discussed this with @Borda in detail and we agreed to have the current members of the platform team as co-codeowners for the pieces outlined in this PR.

While this will certainly not resolve the issues with our development velocity entirely, it will certainly give us the ability to learn more about these areas of the codebase and help in reviewing PRs from other authors, and also help us unblock at least 1 review request without having to chase the current codeowners for the repo. 

I vote to re-open this PR and amend the change to reflect just the current members of the platform team as opposed to listing the entire team in here. @rlizzo will make this change once we're all aligned here. cc: @JatheenA